src/Lib/Lionk.Core.Razor/Attribute/ComponentView.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright © 2024 Lionk Project
+
+namespace Lionk.Core.Component;
+
+/// <summary>
+/// This attribute is used to specify the view that is used to configure the object.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class)]
+public class ComponentView : Attribute
+{
+    /// <summary>
+    /// Gets the views that are used to configure the objects.
+    /// </summary>
+    public static List<ComponentViewModel> ConfigurationViews { get; } = new();
```

Comment:
I believe that having a static field in an attribute is not a good practice.

This list of views should clearly be managed by a service rather than being stored within the attribute itself. Essentially, this is a view registry, and it would be more appropriate to store and manage it within a dedicated service or another component designed for this purpose, rather than in the attribute.

Reply:
@CoJaques  I'm gonna implement this as a service